docs/tech-reference/distributor.rst

```diff
@@ -92,6 +92,43 @@ Supported keys
 
 .. _redirect_file:
 
+V4 Redirect File
+----------------
+
+For Docker v2 content, the distributors generate a json file with the details of the repository
+contents.
+
+The file is JSON formatted with the following keys
+
+* **type** *(string)* - the type of file. This will always be "pulp-docker-redirect".
+* **version** *(int)* - version of the format for the file. For Docker v2, that supports manifest
+                        list, this will be 3.
+* **repository** *(string)* - the name of the repository this file is describing.
+* **repo-registry-id** *(string)* - the name that will be used for this repository in the Docker
+  registry.
+* **url** *(string)* - the URL for accessing the repository content.
+* **schema2_data** *(array)* - an array of tags and digests that schema version 2 image manifests reference.
+* **manifest_list_data** *(array)* - an array of tags and digests that schema version 2 manifest lists  reference.
+* **manifest_list_amd64_tags** *(dict)* - dictionary that contains the information on each manifest list
```

Comment:
Please describe this data structure in enough detail that someone could create valid data by reading this documentation. What exactly should the keys and values be?